docs/hugo/content/reference/breaking-changes.md

```diff
@@ -0,0 +1,434 @@
+
+**Enumerations**: In a few cases we previously had only a base type for a property, without any enumeration values to restrict, even though there were only a few legitimate values to provide. The allowed Kubernetes resource could be created with any value at all even though creation of the Azure resource would fail.
+
+If your resource was using an invalid value for one of these properties, you'll need to modify the YAML for your resource to use a valid value. 
+
+Note that any resource successfully created in Azure will be fine.
+
+**Fixed value fields**: Fields that required a specific value to be provided, with no option; these are similar to Status Only fields.
+
+**Inlined objects**: We're now correctly inlining some objects that were previously generated as nested properties.
+
+If your resource is using one of these, you'll need to modify your resource to promote the property to the parent object.
+
+**Discriminator use**: We now use the discriminator value for polymorphic types to name the property used to specify that option. Previously we were synthesizing a name based on other factors, resulting in longer property names that did not appear in the ARM/Bicep documentation.
+
+If your resource is using one of these, you'll need to change the name of the parent property as detailed below.
+
+## Authorization
+
+### RoleAssignment
+
+| Property | Change (Reason)  | Impact |
+| :------- | :--------------- | :----- |
+| Location | Removed (Unused) | None   |
+| Tags     | Removed (Unused) | None   |
+
+
+## Cache v2020-12-01
+
+### RedisFirewallRule
+
+| Property | Change (Reason)  | Impact |
+| :------- | :--------------- | :----- |
+| Location | Removed (Unused) | None   |
+| Tags     | Removed (Unused) | None   |
+
+### RedisLinkedServer
+
+| Property | Change (Reason)  | Impact |
+| :------- | :--------------- | :----- |
+| Location | Removed (Unused) | None   |
+| Tags     | Removed (Unused) | None   |
+
+### RedisPatchSchedule
+
+| Property | Change (Reason)  | Impact |
+| :------- | :--------------- | :----- |
+| Location | Removed (Unused) | None   |
+| Tags     | Removed (Unused) | None   |
+
+## Cache v2021-03-01
+
+### RedisEnterpriseDatabase
+
+| Property | Change (Reason)  | Impact |
+| :------- | :--------------- | :----- |
+| Location | Removed (Unused) | None   |
+| Tags     | Removed (Unused) | None   |
+
+
+## CDN v2021-06-01
+
+### ProfilesEndpoint
+
+| Property                                                            | Change (Reason)                                       | Impact           |
+| ------------------------------------------------------------------- | ----------------------------------------------------- | ---------------- |
+| DeliveryPolicy.Rules.Actions.DeliveryRuleCacheExpiration            | Renamed to CacheExpiration (discriminator)            | Update your YAML |
```

Comment:
It's more than just "update YAML" - I think they need to mark the resource with `reconcile-policy: skip`, delete it, update their YAML, and then "recreate it" (which will adopt the resource with the same name in Azure).

Reply:
In fact they may need to do that annotation _before_ they upgrade? If they don't, won't we pick the old object up from etc, deserialize it, throw away the field whose name has changed, and apply it (possibly causing issues because we just cleared something they didn't want cleared?)

Reply:
I've added a list of upgrade steps to the doc and expanded the intro.